css/menu.css

```diff
@@ -307,3 +307,7 @@ body.disableScroll {
     height: auto;
   }
 }
+
+.hidden {
```

Comment:
Why is this needed?

![image](https://user-images.githubusercontent.com/290733/109194115-02733500-7791-11eb-84dd-5b010a5afb7e.png)

Bootstrap already has this exact same style.